crates/services/p2p/src/heartbeat/handler.rs

```diff
@@ -73,7 +73,7 @@ impl HeartbeatConfig {
 impl Default for HeartbeatConfig {
     fn default() -> Self {
         Self::new(
-            Duration::from_secs(2),
+            Duration::from_secs(60),
             Duration::from_secs(1),
             NonZeroU32::new(5).expect("5 != 0"),
```

Comment:
It would be nice to make it configurable, but I'm okay with not doing that right now because we already have a lot of configuration for p2p